lib/workers/repository/finalise/index.ts

```diff
@@ -15,4 +17,14 @@ export async function finaliseRepo(
     `Action Required: Fix Renovate Configuration`
   );
   await clearRenovateRefs();
+  const prList = await platform.getPrList();
+  if (
+    prList?.some(
+      (pr) => pr.state === 'merged' && pr.title !== 'Configure Renovate'
+    )
+  ) {
+    logger.debug('Repo is activated');
+    config.repoIsActivated = true;
```

Comment:
SHouldn't we check if the repo has a config file instead, so we can count manually onboarded repos?

Reply:
I think it is not enough to check config file only because we need to make sure that we have at least one merged PR by Renovate and this PR is not the onboarding one.